### DISCUSSION TITLE
ping failing

### DISCUSSION THREAD

**Discussion by djhurlburt** [Navid200/xDrip#189] [Development]
I've been following https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud to the "T" and get to "Phase 2".  The system seems to be trying to ping my subdomain that I created but it continuously fails.  After 12 tries I see it try to contain to 'freedsn.afraid.org' and that seems to work, but then it tries to resume connection to the subdomain and 100% packet loss.  This keeps going 6 more times (gets to trial number 18) and then it once again "resolving freedns.afraid.org" which seems to work, then it once again tries to ping the subdomain several more time and then I get a screen that says:
"Developed by the xDrip team
Please close this window.  Open a new SSH terminal.  Run FreeDNS Setup again to complete FreeDNS setup"

I never get to this step:
After successful setup, you will be asked to enter an email address for security notices.
<pic>
Enter an email address.

I'm not sure if I should create a new subdomain?

**Comment by Navid200** [comment]
These forums are retired.
The forums are now here: https://github.com/NightscoutFoundation/xDrip/discussions
